db/migrate/20200225223106_create_circulation_rules.rb

```diff
@@ -0,0 +1,15 @@
+class CreateCirculationRules < ActiveRecord::Migration[5.2]
+  def change
+    create_table :circulation_rules do |t|
+      t.string :short_code
```

Comment:
For any of your migrations is `short_code` or `name` a "required" field? If so we should `null: false` them